operations/helm/charts/mimir-distributed/templates/query-scheduler/query-scheduler-dep.yaml

```diff
@@ -0,0 +1,132 @@
+{{- if .Values.query_scheduler.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "mimir.resourceName" (dict "ctx" . "component" "query-scheduler") }}
+  labels:
+    {{- include "mimir.labels" (dict "ctx" . "component" "query-scheduler") | nindent 4 }}
+  annotations:
+    {{- toYaml .Values.query_scheduler.annotations | nindent 4 }}
+spec:
+  replicas: {{ .Values.query_scheduler.replicas }}
+  selector:
+    matchLabels:
+      {{- include "mimir.selectorLabels" (dict "ctx" . "component" "query-scheduler") | nindent 6 }}
+  strategy:
+    {{- toYaml .Values.query_scheduler.strategy | nindent 4 }}
+  template:
+    metadata:
+      labels:
+        {{- include "mimir.podLabels" (dict "ctx" . "component" "query-scheduler") | nindent 8 }}
+        {{- with .Values.query_scheduler.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      annotations:
+{{- if .Values.useExternalConfig }}
+        checksum/config: {{ .Values.externalConfigVersion }}
+{{- else }}
+        checksum/config: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
+{{- end}}
+        {{- with .Values.query_scheduler.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
```

Comment:
You should be able to rebase onto main and use 
```
        {{- include "mimir.podAnnotations" (dict "ctx" . "component" "query-scheduler") | nindent 8 }}
```

Reply:
done